dev-client/src/screens/CreateSiteScreen.tsx

```diff
@@ -59,15 +62,28 @@ export const CreateSiteScreen = (props: Props = {}) => {
     [dispatch],
   );
 
+  const onInfo = useCallback(
+    () => infoModalRef.current?.present(),
+    [infoModalRef],
+  );
+  const onInfoClose = useCallback(
+    () => infoModalRef.current?.dismiss(),
+    [infoModalRef],
+  );
+
   return (
-    <ScreenScaffold
-      BottomNavigation={null}
-      AppBar={<AppBar LeftButton={<ScreenCloseButton />} />}>
-      <CreateSiteView
-        createSiteCallback={createSiteCallback}
-        defaultProjectId={'projectId' in props ? props.projectId : undefined}
-        sitePin={'coords' in props ? props.coords : undefined}
-      />
-    </ScreenScaffold>
+    <BottomSheetModalProvider>
+      <ScreenScaffold
+        BottomNavigation={null}
+        AppBar={<AppBar LeftButton={<ScreenCloseButton />} />}>
+        <CreateSiteView
+          createSiteCallback={createSiteCallback}
+          defaultProjectId={'projectId' in props ? props.projectId : undefined}
```

Comment:
question: can you use something like  `defaultProjectId={props?.projectId ?? undefined}`? (same for below)

Reply:
This wasn't my code, but I believe using the 'in' operator in this way checks for the property type, not just the existence. Replacing it with optional chaining raises typescript errors.